CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 2020-11-15, v1.11.0
+
+- Added Provider for Swedish Municipalities
+- Updates to person names in pt_BR
+
```

Comment:
```suggestion
- Styled code according to StyleCI
- Cleaned up README.md (docs are now [here](https://fakerphp.github.io)
```

Reply:
Yeah agree on this as well. Since the docs are really related on how to use the thing. 

I agree that we do not need every small ci and doc fix in the changelog but since we migrated all it is good to add

Reply:
StyleCI was already enabled before this release too.